gui/wxpython/mapdisp/frame.py

```diff
@@ -453,7 +475,7 @@ def AddNviz(self):
         self._giface.WriteCmdLog(
             _("Starting 3D view mode..."), notification=Notification.HIGHLIGHT
         )
-        self.SetStatusText(_("Please wait, loading data..."), 0)
+        self.statusbar.SetStatusText(_("Please wait, loading data..."), 0)
```

Comment:
I suggest to override SetStatusText function (meaning create your own where you call self.statusbar.SetStatusText)